addons/medical/XEH_PREP.hpp

```diff
@@ -1,6 +1,4 @@
 PREP(addDamageToUnit);
-PREP(adjustPainLevel);
```

Comment:
One thing of note in this PR is that I had to move this function to `medical_status` and remove the direct calls to it in `medical_damage` to avoid a dependency (it's now hooked into the `[QEGVAR(medical,injured), [_unit, _painLevel]]` event which is already being raised for screaming).